core/src/main/java/com/linecorp/armeria/server/docs/DocServicePlugin.java

```diff
@@ -31,6 +31,14 @@
  */
 public interface DocServicePlugin {
 
+    /**
+     * Returns the name of this plugin. This method is a shortcut of:
+     * <pre>{@code
+     * getClass().getName();
+     * }</pre>
+     */
```

Comment:
I think there's no point of having this method if this method is a shortcut of `getClass().getName()`. How about returning shorter and simpler names like:

- `grpc`
- `thrift`
- `annotated`